### PR TITLE
Feature/175 dev

### PR DESCRIPTION
**Pull request by joelachance**
Adds a development page in ./thebelab and contribution instructions in README.

I tried `webpack-dev-server`, however, I think the @jupyter package is too large as it doesn't rebuild after changes are made. This seems to work just fine, you simply have to reload the page after making changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,6 +125,26 @@ A full config script with defaults:
...
+After cloning the repository, run:
+
+`npm install`
+
+This will install all dependencies in package.json.
+In order to serve the Thebelab source code, run:
+
+`npm run build:watch`
+
+Now open `development.html` in your root directory to see Thebelab running.
```
Comment (choldgraf) [inline README.md:144]:
```suggestion
Now open `development.html` in your root directory to see Thebelab running. `development.html`
is a simple HTML page that demonstrates Thebelab functionality. You can edit it to test
out new features or configurations.
```

File: README.md [hunk 2]
```diff
@@ -125,6 +125,28 @@ A full config script with defaults:
...
+This will install all dependencies in package.json.
+In order to serve the Thebelab source code, run:
+
+`npm run build:watch`
+
+Now open `development.html` in your root directory to see Thebelab running. `development.html`
+is a simple HTML page that demonstrates Thebelab functionality. You can edit it to test
+out new features or configurations.
+
+This will start Webpack which will bundle and serve the source code. This also will automatically reload re-build changes to the source code, however, you'll be required to refresh the page.
```
Comment (choldgraf) [inline README.md:148]:
```suggestion
Running `npm run` will start Webpack which will bundle and serve the source code
along with `development.html`. As you change the code in Thebelab, the javascript
will automatically be re-built, but you'll be required to refresh the page.
```

**Comment by choldgraf** [comment]
This is super helpful, thanks very much 👍 ah this already makes my life easier haha

**Comment by joelachance** [comment]
Good to hear!

**Pull request merged**